#37: Renamed errorCodeConfig.yml to error_code_config.yml (#38)
exasol/error-code-crawler-maven-plugin: doc/changes/changes_0.4.0.md

Code name:

## Summary

> In this release we renamed to `errorCodeConfig.yml` to `error_code_config.yml` in order to unify the casing of out project files. Don't forget to update the file when updating.

Reviewer: ```suggestion
In this release we renamed `errorCodeConfig.yml` to `error_code_config.yml` in order to unify the casing of our project files. Don't forget to update the file when updating the plugin version.
```

## Refactoring

> * #35: Remove hardcoded version numbers from tests

Reviewer: ```suggestion
* #35: Removed hardcoded version numbers from tests
```

> * #37: Renamed errorCodeConfig.yml to error_code_config.yml

Reviewer: ```suggestion
* #37: Renamed `errorCodeConfig.yml` to `error_code_config.yml`
```
